test/UnitTestParquetCpp.exec.out.tmp

```diff
@@ -0,0 +1,2 @@
+ld: warning: directory not found for option '-L/opt/homebrew/Caskroom/miniforge/base/envs/ak-base/lib'
+timedexec: target program died with signal 11, without coredump
```

Comment:
We prob don't want this file

Reply:
Removed. I ran `git add test/` and did not realize that was in there.